docs/releasing/testing-and-linting.md

```diff
@@ -47,3 +47,23 @@ If a snapshot test fails, review the difference in the console. If the change is
 `npm test -- -u src/govuk/components/button`
 
 This will update the snapshot file. Commit this file separately with a commit message that explains you're updating the snapshot file and an explanation of what caused the change.
+
+## Visual regression testing with Percy
+
+We use [Percy](https://percy.io/), a visual regression testing tool, to generate and store screenshots of our components to help us check for any unintended visual changes.
+
+We generate two screenshots for each default example of every component: one with JavaScript enabled and one with JavaScript disabled. Screenshots are not taken for all the different variations of each component. This tool is not a replacement for manual testing.
+
+The screenshots are public, so they can be checked without logging in. A BrowserStack account is needed to approve or reject any changes. It's the responsibility of the person reviewing the pull request code to approve any visual changes that Percy highlights.
```

Comment:
```suggestion
The screenshots are public, so you can check them without logging in. A BrowserStack account is needed to approve or reject any changes. If you're the reviewer of the pull request code, it's your responsibility to approve any visual changes Percy highlights.
```

Does the user need to have a BrowserStack account? If they do, do we need to link to BrowserStack signup?

Reply:
@EoinShaughnessy They need to have a BrowserStack account to approve or reject changes. I'm not sure about adding a link to BrowserStack signup - I think an existing member needs to add them to the Design System team. Maybe I'll add a line about asking the tech lead in the team to get you access...